Sekoia.io/sekoiaio/operation_center/__init__.py

```diff
@@ -297,6 +297,15 @@
     },
 )
 
+AddEventsToACase = type(
+    "AddEventsToACase",
+    (GenericAPIAction,),
+    {
+        "verb": "post",
+        "endpoint": base_url + "cases/{case_uuid}/events",
```

Comment:
The variable must have the same name as in the action declaration.
```suggestion
        "endpoint": base_url + "cases/{uuid}/events",
```